src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -2909,12 +2896,7 @@ PaSampleFormat WaveToPaFormat(const WAVEFORMATEXTENSIBLE *fmtext)
             }
         }
         else
-        if (IsEqualGUID(&fmtext->SubFormat, &pa_KSDATAFORMAT_SUBTYPE_IEC61937_DOLBY_DIGITAL))
-        {
-            return paInt16;
-        }
-        else
-        if (IsEqualGUID(&fmtext->SubFormat, &pa_KSDATAFORMAT_SUBTYPE_IEC61937_DOLBY_DIGITAL_PLUS))
+        if (IsEqualGUID(&fmtext->SubFormat, &pa_KSDATAFORMAT_SUBTYPE_IEC61937_PCM))
```

Comment:
@dmitrykos The logic of comparing GUIDs here is wrong and will cause all `fmtext->SubFormat` that are not `pa_KSDATAFORMAT_SUBTYPE_IEC61937_PCM` to return paCustomFormat. After this function correctly returns `paInt16`, pa_test is played normally.